packages/package-values.yaml

```diff
@@ -39,7 +39,7 @@ managementPackageRepository:
     - name: addons-manager
```

Comment:
We probably need to change this name to v2 as well. It's the name shown in buildinfo file.

Reply:
udpated the names